### ISSUE TITLE
Usability issue with pdb

### ISSUE THREAD

**Issue by lelit** [Pylons/hupper#14]
When I run my application with `hupper -m` and hit a `pdb.set_trace()`, the prompt appears and I can interact with the debugger, but neither cursors nor tab-completion work as expected. Instead they emit their raw escape sequence, for example this is what happens when I press the _cursor up_ key:

```
(Pdb) l
…
 10  	def foo():
 11  	    import pdb; pdb.set_trace()
 12  ->	    pass
[EOF]
(Pdb) ^[[A
```

Running it without hupper does not exhibit the problem.

This is on GNU/Linux (Debian sid), with Python 3.6.

**Comment by mmerickel** [comment]
Hey thanks for the report. I probably need someone to tell me the right way to do this - I'm not familiar with handling terminal escape sequences in a subprocess.

**Comment by mmerickel** [comment]
This looks like an issue with readline but I'm not sure what it is yet. What I do know is that the subprocess properly thinks stdin is a tty and that ipdb works fine but that pdb does not.

**Comment by mmerickel** [comment]
@bertjwregeer figured out what was going on here... the fix is in #15 and will be released soon.

**Comment by lelit** [comment]
Thank you!
